### PR TITLE
Create setters example

### PR DESCRIPTION
**Pull request by phanimarupaka**
This PR is to add an e2e example for creating setters using `search-replace` function.

### CODE REVIEW

File: examples/search-replace/create-setters/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  replicas: 3
+  template:
+    spec:
+      containers:
+        - name: nginx
+          image: "gcr.io/nginx:1.14.2"
+```
+
+Suppose you want to expose the values of `replicas`, `image` and `tag` as parameters.
+You can create setters by invoking `search-replace` function with following arguments:
```
Comment (frankfarzan) [inline examples/search-replace/create-setters/README.md:25]: cross link to "setters", don't assume it's universal knowledge

File: examples/search-replace/create-setters/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+```sh
+'by-path=spec.replicas' 'put-comment=kpt-set: ${replicas}'
```
Comment (frankfarzan) [inline examples/search-replace/create-setters/README.md:28]: Why not show the kpt CLI (you show the CLI below anyways)?

Comment (frankfarzan) [inline examples/search-replace/create-setters/README.md:28]: Shouldn't only the value be quoted?

File: examples/search-replace/create-setters/README.md (hunk 1 above)
Comment (phanimarupaka) [reply examples/search-replace/create-setters/README.md]: Updated.

File: examples/search-replace/create-setters/README.md (hunk 2 above)
Comment (phanimarupaka) [reply examples/search-replace/create-setters/README.md]: Done.

Comment (phanimarupaka) [reply examples/search-replace/create-setters/README.md]: In general it is better to quote entire key=value as keys may also have regex characters.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**